infrastructure/httpapi/handlers/bridges.go

```diff
@@ -366,3 +396,24 @@ func parseStatus(value string) (*types.Status, error) {
 		return nil, errors.New("unrecognized status")
 	}
 }
+
+func DefaultCronosEVMAddressHookGenerator(accountAddressPrefix string) func(string) (string, error) {
```

Comment:
Could we put this kind of address parser in a separated package? e.g. bridges/cronos, bridges/parsers/cronos

So that to start to make the bridge activity be more generic

Reply:
good suggstion

Reply:
Updated, I moved it under the `external` package, wdyt?

Reply:
yup, lets do in `external` first